CHANGELOG.md

```diff
@@ -5,6 +5,11 @@
 ### Added
 - Your feature here!
 
+## [1.4.1] - 2017-04-18
+
+### Added
+- Added `helperResolver` config option to override the default helper resolution
```

Comment:
I would just put this under the `[Unreleased]` section, and I'll update with a version number and date when I get around to making a release (which will hopefully be soon).